core/src/main/kotlin/no/ntnu/beardblaster/utils/AbstractScreen.kt

```diff
@@ -1,12 +1,18 @@
-package no.ntnu.beardblaster
+package no.ntnu.beardblaster.utils
 
 import com.badlogic.gdx.graphics.g2d.Batch
 import ktx.app.KtxScreen
+import no.ntnu.beardblaster.BeardBlasterGame
+
+
+interface IBeardBlasterScreen {
+    fun setBtnEventListeners()
+}
 
 abstract class AbstractScreen(
         val game: BeardBlasterGame,
         val batch: Batch = game.batch
-) : KtxScreen {
+) : KtxScreen, IBeardBlasterScreen {
```

Comment:
Nå må alle screens override "setBtnEventListeners" selv om de ikke implementerer noen knapper. 
Et forslag er at man heller selv implementerer dette interfacet dersom man behøver buttonlisteners.

Kanskje kalle interfacet `ButtonEventListener`? 


Klassen for LoginScreen ville sett slik ut:

```java

class GameplayScreen(game: BeardBlasterGame) : AbstractScreen(game), ButtonEventListener { 
 // class-impl
 }
```

Reply:
er egentlig litt usikker på hvor god den metoden der er heller - man må liksom uansett implementere et interface som gir deg én metode, som du selv også må kalle på..